base/cvd/cuttlefish/host/commands/cvd/selector/instance_database.cpp

```diff
@@ -301,5 +302,26 @@ Result<bool> InstanceDatabase::GetAcloudTranslatorOptout() const {
       });
 }
 
+Result<std::string> InstanceDatabase::GenUniqueGroupName() const {
```

Comment:
Related to the comment in `instance_manager.cpp`, should this be worked into the implementation of `AddInstance`, that if a group name is missing it will generate a guaranteed unique one?

Reply:
Done